.github/workflows/continuous-integration.yml

```
@@ -205,16 +205,26 @@ jobs:
         id: get-broken-link-info
         run: node ./script/github-actions/check-broken-links.js ${{ matrix.buildtype }}
 
+      - name: Generate build details
+        run: |
+          cat > build/${{ matrix.buildtype }}/BUILD.txt << EOF
+          BUILDTYPE=${{ matrix.buildtype }}
+          BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}")
+          RUN_ID=${{ github.run_id }}
+          REF=${{ github.sha }}
+          BUILDTIME=$(date +%s)
+          EOF
+
       - name: Compress ${{ matrix.buildtype }} build
         if: ${{ github.ref == 'refs/heads/master' && success() }}
-        run: tar -zcvf ${{ matrix.buildType }}.tar.gz build
+        run: tar -jcvf ${{ matrix.buildType }}.tar.bz2 build/${{ matrix.buildType }}
 
       - name: Upload build
         uses: actions/upload-artifact@v2
         if: ${{ github.ref == 'refs/heads/master' && success() }}
         with:
           name: ${{ matrix.buildtype }}
-          path: ${{github.workspace}}/content-build/${{ matrix.buildType }}.tar.gz
+          path: ${{ github.workspace }}/content-build/${{ matrix.buildType }}.tar.bz2
```

I'm inclined to use relative paths from the working directory where possible. I think that would help narrow down the scope in our own mental models when thinking about the operations taking place. Like this could just be:

```suggestion
          path: ${{ matrix.buildType }}.tar.bz2
```